File: smallBall/MoveImage/ViewController.m
//
//  ViewController.m
//  MoveImage
//
//  Created by 商 文博 on 12-12-2.
//  Copyright (c) 2012年 商 文博. All rights reserved.
//

#import "ViewController.h"
#import "BlueViewController.h"
#import "YeallowViewController.h"

@interface ViewController ()

@end

@implementation ViewController
@synthesize blueViewController;
@synthesize yeallowViewController;

- (void)viewDidLoad
{
    BlueViewController *blueTemp=[[BlueViewController alloc] initWithNibName:@"BlueViewController"bundle:nil];
	self.blueViewController = blueTemp;
	[self.view insertSubview:self.blueViewController.view atIndex:0];//0表示再最顶层显示；
	
	[blueTemp release];
    [super viewDidLoad];
}
-(IBAction)switchView:(id)sender
{
	if(self.blueViewController.view.superview == nil)
	{
            self.blueViewController=[[BlueViewController alloc] initWithNibName:@"BlueViewController"bundle:nil];
		[self.yeallowViewController.view removeFromSuperview];
		[self.view insertSubview:self.blueViewController.view atIndex:0];//0表示再最顶层显示；
	}
	else
        if(self.yeallowViewController.view.superview == nil)
        {
                self.yeallowViewController=[[YeallowViewController alloc] initWithNibName:@"YeallowViewController"bundle:nil];
            [self.blueViewController.view removeFromSuperview];
            [self.view insertSubview:self.yeallowViewController.view atIndex:0];
        }
}


- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
}

@end

File: Fun iOS App/Fun iOS App/ViewController.swift
//
//  ViewController.swift
//  Fun iOS App
//
//  Created by Nancy Zhang on 13/08/17.
//  Copyright © 2017 Nancy Zhang. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    
    @IBOutlet weak var theLabel: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func clickingTheButton(_ sender: Any) {
        
        theLabel.text = "Clicked button"
        print("Hello, world!")
        
        let courseNameList = ["Christine", "Justin", "Adam", "Greg"]
        
        for name in courseNameList
        {
            if name == "Christine"
            {
                print("\(name) is a lecturer")
            }
            else
            {
                print("\(name) is a student")
            }
        }

    }

}

